### PR TITLE
Adding Oodrive Sign connector (certified-connectors)

### PR DESCRIPTION
**Pull request by QwerioTEAM**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by QwerioTEAM** [comment]
@microsoft-github-policy-service agree company="Qwerio"

I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2891 @@
...
+            "description": "List contracts only in a given perimeter.",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "status",
+            "in": "query",
+            "x-ms-summary": "Only with status",
+            "type": "array",
+            "collectionFormat": "csv",
+            "description": "Retrieve contract only with specified status",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:118]: Hi @QwerioTEAM, Thank you so much for your submission. 
All parameters descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix this for all the operations

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2891 @@
...
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/Transaction"
+            }
+          }
+        },
+        "summary": "Get a contract's status",
+        "description": "Check a specific contract's status",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:147]: @QwerioTEAM all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please fix this for all the operations in the swagger file

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2891 @@
...
+              "type": "file",
+              "title": "PDF Document Binary"
+            }
+          }
+        },
+        "summary": "Download the contract's documents",
+        "produces": [
+          "application/pdf"
+        ],
+        "description": "Retrieve a PDF file that contains all the documents from a contrat",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:661]: @QwerioTEAM all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations), and all file should be free of grammatical errors or typos. 
here you have "contrat" please fix to "contract"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2891 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Specify if the notification was successfuly resend"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1231]: @QwerioTEAM all files should be free of grammatical errors. please change "successfuly" to "successfully"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2891 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Specifiy if the notification was successfuly resend"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1275]: @QwerioTEAM all files should be free of grammatical errors. please change "Specifiy" to "Specify" and "successfuly" to "successfully"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2891 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Specify if it was removed successfuly"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1365]: @QwerioTEAM all files should be free of grammatical errors. please change "successfuly" to "successfully"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,2891 @@
...
+              "type": "file"
+            }
+          }
+        }
+      }
+    },
+    "/contracts/recipients/{cfc_id}": {
+      "delete": {
+        "summary": "Delete a recipient for contract",
+        "description": "Delete a recipient from a contract, by his recipient for contract ID. More information in the documentaton.",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1344]: @QwerioTEAM all files should be free of grammatical errors. please change "documentaton" to "documentation"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,2891 @@
...
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "succeed": {
+                  "type": "boolean",
+                  "description": "Says if bundle was withdrawed"
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1599]: @QwerioTEAM all files should be free of grammatical errors. please change "withdrawed" to "withdrawn"

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,2891 @@
...
+        },
+        "summary": "List all perimeters",
+        "description": "Retrieve all perimeters associated to the connected environment",
+        "operationId": "ListAllPerimeters"
+      }
+    },
+    "/recipients": {
+      "get": {
+        "summary": "List all recipients",
+        "description": "Retrieve all the recipients associated to the current licence",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:1766]: @QwerioTEAM all files should be free of grammatical errors. please change "licence" to "license"

File: certified-connectors/Oodrive Sign/apiProperties.json [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        "title": "AcceptJsonHeader",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "Accept",
+          "x-ms-apimTemplateParameter.value": "application/json",
+          "x-ms-apimTemplateParameter.existsAction": "skip",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "Qwerio",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiProperties.json:32]: @QwerioTEAM is it possible to have the company name at the publisher?

File: certified-connectors/Oodrive Sign/readme.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
+# Oodrive Sign Connector
+The Sign solution from Oodrive helps organizations to easily manage, prepare, sign and share agreements. This connector connects to your Oodrive Sign environment and offers functionalities as you can retrieve in the web solution. 
+
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/readme.md:3]: @QwerioTEAM please add all the required section in the readme file. you can see our template [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/certified-connectors)

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 12]
```diff
@@ -0,0 +1,2891 @@
...
+                },
+                "endpoint": {
+                  "type": "object",
+                  "properties": {
+                    "url": {
+                      "type": "string",
+                      "description": "url",
+                      "x-ms-notification-url": true,
+                      "x-ms-visibility": "internal",
+                      "title": ""
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:2060]: @QwerioTEAM empty property is not allowed, please define the title.

File: certified-connectors/Oodrive Sign/apiDefinition.swagger.json [hunk 13]
```diff
@@ -0,0 +1,2891 @@
...
+        "birthdate": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Birth date (timestamp) of the recipient.",
+          "x-ms-summary": "Birthdate"
+        },
+        "birthplace": {
+          "type": "string",
+          "description": "Birth place of the recipient.",
+          "x-ms-summary": ""
```
Comment (Amjed-Ayoub) [inline certified-connectors/Oodrive Sign/apiDefinition.swagger.json:2759]: @QwerioTEAM empty property is not allowed, please define the x-ms-summary.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
@QwerioTEAM Thank you so much for the fix. May I ask you please to update the readme file as well. Please resolve all the conversations.

**Comment by Amjed-Ayoub** [comment]
Hello @QwerioTEAM,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by QwerioTEAM** [comment]
> Hello @QwerioTEAM, I hope you are doing well. I am following up with you to see how things are going with preparing your connector. Please let us know if you have any questions. Thank you.

Hello @Amjed-Ayoub 
We have a lack of time currently to get the readme done. We're expecting it to be done next week. I'll let you know about our progress in the process (describing all operations contained in the connector is quite time-consuming actually).

**Comment by Amjed-Ayoub** [comment]
Hi @QwerioTEAM please let me know when this PR is ready for review.

**Comment by QwerioTEAM** [comment]
> Hi @QwerioTEAM please let me know when this PR is ready for review.

Hello @Amjed-Ayoub 
It seems OK on our side for a review. Let me know if anything needs to be changed.

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @QwerioTEAM,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**